enhancements/agent-installer/agent-based-installer.md

```diff
@@ -0,0 +1,279 @@
+---
+title: agent-based-installer
+authors:
+  - "@zaneb"
+reviewers:
+  - "@avishayt"
+  - "@celebdor"
+  - "@hardys"
+  - "@lranjbar"
+  - "@mhrivnak"
+  - "@patrickdillon"
+  - "@pawanpinjarkar"
+  - "@romfreiman"
+  - "@rwsu"
+  - "@sdodson"
+approvers:
+  - "@dhellmann"
+creation-date: 2022-03-03
+last-updated: 2022-03-03
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/AGENT-9
+---
+
+# Agent-based Installer
+
+## Summary
+
+The agent-based installer is an installation method that will use a bootable,
```

Comment:
It's covered well in the rest of the doc but it would be good to mention in the summary that the current focus is on-prem platforms.